extensions/ql-vscode/src/remote-queries/view/ExportButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+const Button = styled.a`
+  color: var(--vscode-button-foreground);
+  background-color: var(--vscode-button-background);
+  &:hover {
+    color: var(--vscode-button-foreground);
```

Comment:
Is this right? It's the same class as non-hover mode. Did you mean this?

```suggestion
    color: var(--vscode-button-hoverForeground);
```